### ISSUE TITLE
Travis Build Fails Randomly (Because of C++ Microsteps test?)

### ISSUE THREAD

**Issue by mew2ub** [icyphy/lingua-franca#148]
I just pushed a commit in which I added a new benchmark, triggering a travis build. The build failed and I'm pretty confident my change didn't affect anything. I've noticed this happen several times. I'll push an unrelated change, the build fails, I restart the build and it passes. Has anyone else noticed this?

On this particular run it looks like the C++ test for Microsteps is the culprit. Is there any reason for that test to fail non deterministically?

**Comment by cmnrd** [comment]
Yes, there apears to be some problem in the C++ build process (it is the build that failed, not the actual test program). I have observed this particular issue several times now. The C++ target has an external dependency on CLI11. It is a single header library and I configured cmake to simply download the header file and place it in the build directory. I assume that it is the download that fails sometimes. I plan to place the CLI11 header in the lib directory of LF instead and copy it from there (the license of CLI11 allows redistribution). Hopefully this solves the issue.

I also observed [AsyncCallbackDrop](https://github.com/icyphy/lingua-franca/blob/master/test/C/AsyncCallbackDrop.lf) fails sometimes and than works on the next commit.   I wasn't able to find a failed build log though.
